test/ci/ci_tests.cfg

```
@@ -502,14 +545,61 @@ args=%(STDARGS_ICARUSCODE)s --input-file %(INPUT_STREAM)s --reference-files %(RE
 parse_art_output=True
 output1=%(TFILENAME)s
 
+[test single_reco0_seq_test_icaruscode]
+STAGE_NAME=reco0
+INPUT_STAGE_NAME=detsim
+NEVENTS=%(NEVENTS_SEQ_SINGLE_ICARUSCODE)s
+# calibrated with [e20] on icarusbuild02.fnal.gov
+cpu_usage_range=100:402
+mem_usage_range=700000:1200002
+
+script=%(EXPSCRIPT_ICARUSCODE)s
+requires=single_%(INPUT_STAGE_NAME)s_seq_test_icaruscode
+FHiCL_FILE=%(CI_FHICL_PREFIX_ICARUSCODE)ssingle_%(STAGE_NAME)s_seq_test_icaruscode.fcl
+INPUT_BASE_FILE_NAME=single_%(INPUT_STAGE_NAME)s_test_icaruscode
+OUTPUT_BASE_FILE_NAME=single_%(STAGE_NAME)s_test_icaruscode
+EXTRA_DIR=%(SINGLE_INPUT_DIRNAME_ICARUSCODE)s
+INPUT_STREAM=../%(requires)s/%(INPUT_BASE_FILE_NAME)s_%(CUR_ICARUSCODE)s.root
+#outBNB is added to specify the output and might need to be removed later
+OUTPUT_STREAM=outBNB:%(OUTPUT_BASE_FILE_NAME)s_%(CUR_ICARUSCODE)s.root 
+REFERENCE_FILES=%(XROOTD_REFERENCEFILEDIR_ICARUSCODE)s/%(EXTRA_DIR)s/%(STAGE_NAME)s/%(OUTPUT_BASE_FILE_NAME)s_%(REF_ICARUSCODE)s.root
+TFILENAME=hist-%(OUTPUT_BASE_FILE_NAME)s.root
+args=%(STDARGS_ICARUSCODE)s --input-file %(INPUT_STREAM)s --reference-files %(REFERENCE_FILES)s --extra-options --TFileName,%(TFILENAME)s
+parse_art_output=True
+output1=%(TFILENAME)s
+
+[test single_reco1_seq_test_icaruscode]
+STAGE_NAME=reco1
+INPUT_STAGE_NAME=reco0
+NEVENTS=%(NEVENTS_SEQ_SINGLE_ICARUSCODE)s
+# calibrated on icarusbuild01.fnal.gov on 20180325 (same as quick test because detsim has reduced number of events)
+# calibrated with [e19] on icarusbuild02.fnal.gov on 20200921
+# calibrated with [e20] on icarusbuild02.fnal.gov on 20210607: 800 MB -> 3.8 GB!!!
+cpu_usage_range=100:402
+mem_usage_range=700000:1200002
```

Consider using a marking different than `2`:
```suggestion
cpu_usage_range=100:406
mem_usage_range=700000:1200006
```
or whatever is available.
As a reminder, those weird numbers are there just to facilitate lookup in case of failure: the message tells you that resource usage was outside the limit `700000:1200006` and you can look up the configuration easily by searching for `0006` instead of the test name (which you may need to copy and paste).